test/Tryouts/Program.cs

```diff
@@ -118,6 +125,13 @@ public unsafe static void Main(string[] args)
                         {
                             using (ctx.OpenReadTransaction())
                             {
+                                if (i == 0)
```

Comment:
Let's start adding real tests

Reply:
I will move this to a separate project then